src/exprs.c

```diff
@@ -919,9 +919,19 @@ Obj             EvalPermExpr (
 **
 **  'EvalListExpr'  just  calls 'ListExpr1'  and  'ListExpr2' to evaluate the
 **  list expression.
+**
+**  The 'safe' argument to ListExpr1 and ListExpr2 should be set to 1 when
+**  the list expression may be modified when its elements are being
```

Comment:
"when its elements are being constructed" -> "while its ..." ? (It took me a moment to parse the current phrasing, but then I did understand it, so it's probably fine as it is -- I just try to be super sensitive and try to simulate how I might read a paragraph if I knew less about the kernel; but of course that's imperfect. Perhaps e.g. @ssiccha would be willing to chime and say whether the above text seems clear to him -- hopefull it is, but  if not, it'd be useful to know which bits are not.

Reply:
@fingolfin @ChrisJefferson Apart from knowing what exactly an expression is (`system.h` says its a `UInt`, but that's all I found out quickly), the formulation is clear to me.

Reply:
> "when its elements are being constructed" -> "while its ..." 
"while its" also sounds better to me